.github/workflows/stale.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 */6 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          # General rules applied to both, issues and pull requests (PRs)
+          start-date: "2022-07-30T00:00:00Z"
+          days-before-close: 14
+          days-before-stale: 90
+          exempt-all-assignees: true
+          exempt-all-milestones: true
+
+          # Rules applied to Issues
+          exempt-issue-labels: "Epic, bug, priority-critical, priority-high, community-upvoted, for-review, security, keep"
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had
+            recent activity. It will be closed if no further activity occurs.
+          close-issue-message: >
+            This issue has been automatically closed due to lack of activity. In an
+            effort to reduce noise, please do not comment any further.
```

Comment:
We'd allow re-opening though, right?

Reply:
Yes, I will prepare updated text.

Reply:
The text now reflects that reopening is ok.